src/modules/display.js

```diff
@@ -1,18 +1,64 @@
-const scores = [
-  { Name: 'John', score: 100 },
-  { Name: 'kioko', score: 79 },
-  { Name: 'Mutisya', score: 70 },
-  { Name: 'Engineer', score: 85 },
-];
+/* eslint-disable */
+const userName = document.querySelector('.name-input');
+const userScore = document.querySelector('.score-input');
+const submitBtn = document.querySelector('.form-submit');
+const refreshBtn = document.querySelector('.refresh');
+const endpoint = new URL(
+  'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/WiYu2SZ3z9RSy8b97AHH/scores',
+);
 
-const scoresData = document.getElementById('user-data');
+const postData = async () => {
+  const name = userName.value;
+  const score = userScore.value;
 
-let userData = '';
+  if (!name) {
+    return;
+  }
 
-scores.forEach((score) => {
-  userData += `
-  <h4 class="user-score">${score.Name} : ${score.score}</h4>
-`;
+  const response = await fetch(endpoint, {
+    method: 'POST',
+    headers: {
+      Accept: 'application/json',
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify({
+      name: 'leaderboard-game',
+      user: name,
+      score: Number(score),
+    }),
+  });
+};
+
+const loadData = async () => {
+  const userScores = document.getElementById('user-data');
+
+  const response = await fetch(endpoint);
+
+  const data = await response.json();
+
+  Object.entries(data).map((entry) => {
+    const value = entry[1];
+    let userData = '';
+
+    value.forEach((e) => {
```

Comment:
- [**Optional**] To avoid a random display of the scores on the board, we can sort it:

`value.sort((a, b) => b.score - a.score).forEach((e) => {`

Reply:
Thanks. I have implented that and it looks more cleaner now.